src/SfxWeb/src/app/modules/event-store/event-store.module.ts

```diff
@@ -23,6 +23,6 @@ import { ConcurrentEventsVisualizationModule } from '../concurrent-events-visual
     NgbTooltipModule,
     ConcurrentEventsVisualizationModule
   ],
-  exports: [EventStoreComponent, FullDescriptionComponent, RowDisplayComponent, OptionPickerComponent],
+  exports: [EventStoreComponent, FullDescriptionComponent, RowDisplayComponent, OptionPickerComponent, DoubleSliderComponent],
```

Comment:
this import should go away when rebased to use the new event store wrapper component.

Reply:
I still need to export the new TimePicker component here